### DISCUSSION TITLE
OrganizationSwitcher UI suggestions

### DISCUSSION THREAD

**Discussion by nick-kang** [clerk/javascript#1964] [Feature Requests]
### Preliminary Checks

- [X] I have reviewed https://clerk.com/docs for existing features that would solve my problem

- [X] I have searched for existing feature requests: https://github.com/clerkinc/javascript/issues

- [X] This issue is not a question, general help request, or anything other than a feature request directly related to Clerk. Please ask questions in our Discord community: https://clerk.com/discord.

### Description

Noticed a couple spots the switcher can be improved. Feel free to close if this isn't the correct forum to post something like this.

* Remove grey divider under "Manage Organization" when there's nothing to display under that divider. 
  <img width="393" alt="Screenshot 2023-10-04 at 7 13 08 PM" src="https://github.com/clerkinc/javascript/assets/89998043/6095c192-1e9d-4ecb-9b93-cf73c1257f60">

* ~~Vertically center the switcher `<button/>` inside the parent `<div/>`.~~ Add `display: inline-flex` to parent `<div/>` 
  <img width="393" alt="Screenshot 2023-10-04 at 7 12 39 PM" src="https://github.com/clerkinc/javascript/assets/89998043/3afdec93-762c-40a2-b558-01969cce3e12">

### Affected Packages

@clerk/clerk-js

**Comment by Alone-Y154** [comment]
Hi, I would like to work on it, can you assign it to me

**Comment by jescalan** [comment]
@Alone-Y154 - you got it! appreciate the willingness to jump in. Please let us know if you need any help/guidance 😁
